### PR TITLE
added user nickname changing

### PR DESCRIPTION
**Pull request by Leminode**
only for nicknames right now, changing the email field wont do anything;
there is also a problem with user details not refreshing after updating them on auth0

### CODE REVIEW

File: Energy Saver/Pages/Account/Profile.cshtml [hunk 1]
```diff
@@ -14,10 +14,21 @@
                      alt="" class="img-rounded img-responsive" />
             </div>
             <div class="col-md-4 text-color">
-                <h3>@Model.UserName</h3>
-                <p>
-                    <i class="glyphicon glyphicon-envelope text-color"></i> @Model.UserEmailAddress
-                </p>
+                <form method="post">
+                    <div class="form-group">
+                        <label asp-for="UserName" class="control-label">User Name</label>
```
Comment (Tauras-Rim) [inline Energy Saver/Pages/Account/Profile.cshtml:19]: There could be a button for the option to edit, not editable fields on page load

**Review by Tauras-Rim — COMMENTED** [review]
For some reason when I change the user name field it reverts back to my old user name and shows a success notification. Not sure if this works

**Comment by Tauras-Rim** [comment]
FIXED. The Auth0ApiToken was missing

**Review by Tauras-Rim — APPROVED** [review]

File: Energy Saver/Pages/Account/Profile.cshtml (hunk 1 above)
Comment (Leminode) [reply Energy Saver/Pages/Account/Profile.cshtml:19]: no

**Review by priolette — APPROVED** [review]

**Pull request merged**